TimeService/work/timeservice/implem/default/C/src/timeservice-rtems.h

```diff
@@ -0,0 +1,46 @@
+#ifndef TIMESERVICE_RTEMS6_H
+#define TIMESERVICE_RTEMS6_H
+
+#include "interfaces_info.h"
+
+#define TICKS_PER_SECOND 1000000
+#define SCALER_VALUE 5
+
+static uint64_t get_time()
+{
+   uint64_t ticks = UINT64_MAX - ((uint64_t)(Timer_Apbctrl1_getCounterValue(&timer_2) + 1) << 32
```

Comment:
I am not really sure whether this checks out. Are we using the system timer? If so, it ticks ~100 or 1000 per second, which is way to low if we want nanosecond precision, as required.

Reply:
I added a comment explaining that the line reuses perf-mon timers. The timer ticks ~1'000'000 times per second.